src/globus_sdk/services/auth/client/service_client.py

```diff
@@ -631,3 +680,1187 @@ def delete_project(self, project_id: UUIDLike) -> GlobusHTTPResponse:
+
+    def delete_policy(self, policy_id: UUIDLike) -> GlobusHTTPResponse:
+        """
+        Delete a policy. Requires the ``manage_projects`` scope.
+
+        :param policy_id: The ID of the policy to delete
+        :type policy_id: str or uuid
+
+        .. tab-set::
+
+            .. tab-item:: Example Usage
+
+                .. code-block:: pycon
+
+                    >>> ac = globus_sdk.AuthClient(...)
+                    >>> policy_id = ...
+                    >>> r = ac.delete_policy(policy_id)
+
+            .. tab-item:: Example Response Data
+
+                .. expandtestfixture:: auth.delete_policy
+
+            .. tab-item:: API Info
+
+                ``DELETE /v2/api/policies/{policy_id}``
+
+                .. extdoclink:: Delete Policy
+                    :ref: auth/reference/#delete_policy
+        """
+        return self.delete(f"/v2/api/policies/{policy_id}")
+
+    def get_client(
+        self,
+        *,
+        client_id: UUIDLike | None = None,
+        fqdn: str | None = None,
+    ) -> GlobusHTTPResponse:
+        """
+        Look up a client by ``client_id`` or (exclusive) by ``fqdn``.
+        Requires the ``manage_projects`` scope.
+
+        :param client_id: The ID of the client to look up
+        :type client_id: str or uuid
+        :param fqdn: The fully-qualified domain name of the client to look up
+        :type fqdn: str
+
+        .. tab-set::
+
+            .. tab-item:: Example Usage
+
+                .. code-block:: pycon
+
+                    >>> ac = globus_sdk.AuthClient(...)
+                    >>> # by client_id
+                    >>> r = ac.get_client(client_id="6336437e-37e8-4559-82a8-674390c1fd2e")
+                    >>> r.data
+                    {
+                      'client": {
+                        'required_idp": None,
+                        'name": "Great client of FOO",
+                        'redirect_uris": [],
+                        'links": {
+                          'privacy_policy": None,
+                          'terms_and_conditions": None
+                        },
+                        'scopes": [],
+                        'grant_types": [
+                          'authorization_code",
+                          'client_credentials",
+                          'refresh_token"
+                        ],
+                        'id": "6336437e-37e8-4559-82a8-674390c1fd2e",
+                        'prompt_for_named_grant": False,
+                        'fqdns": ["globus.org"],
+                        'project": "da84e531-1afb-43cb-8c87-135ab580516a",
+                        'client_type": "client_identity",
+                        'visibility": "private",
+                        'parent_client": None,
+                        'userinfo_from_effective_identity": True,
+                        'preselect_idp": None,
+                        'public_client": False
```

Comment:
Just spot-checking, I see this one has `'public_client": False` -- note the mismatched quote chars. Other keys fit that pattern too.
I'm guessing that this came from some python with double-quotes, and then it went through `s/"/'/` when you meant `s/"/'/g`. "Classic" sed typo. :smile: 

Anyway, I'm calling this one out for early feedback, but we should double-check the others.

Reply:
👍 I actually saw this in `get_projects()` and assumed that the ' followed by " was required by some doc generator. I'll change this throughout the file.

Reply:
Oh, so it's _my_ typo! Sorry about that. Yeah, we should fix them all, but you only have to worry about your new ones -- I'll circle back for anything pre-existing.